cache/src/main/kotlin/com/dropbox/android/external/cache4/RealCache.kt

```diff
@@ -78,6 +81,8 @@ internal class RealCache<Key : Any, Value : Any>(
      */
     private val loadersSynchronizer = KeyedSynchronizer<Key>()
 
+    private var totalWeight = 0L
```

Comment:
was this reverted?

Reply:
Oops good catch. I'll add tests for the new builder as well. BTW your tests were great and caught a logic error of mine